fiftyone/utils/iou.py

```
@@ -624,6 +685,61 @@ def _compute_bbox_ious(
+    if sparse:
+        ious = defaultdict(list)
+    else:
+        ious = np.zeros((len(preds), len(gts)))
+
+    if iscrowd is not None:
+        gt_crowds = [iscrowd(gt) for gt in gts]
+    else:
+        gt_crowds = [False] * len(gts)
+
+    index_property = rti.Property()
+    bbox_iou_fcn = compute_bbox_iou
```

Ruff (0.8.2)

Local variable `bbox_iou_fcn` is assigned to but never used

Remove assignment to unused variable `bbox_iou_fcn`

(F841)